### PR TITLE
[rush] Support "." as a value for the --to and --from parameters to build to the current project.

### PR DESCRIPTION
**Pull request by iclanton**

### CODE REVIEW

**Comment by sachinjoseph** [comment]
I've always wanted to be able to do this! 😍

File: apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts [hunk 1]
```diff
@@ -180,7 +185,8 @@ export class BulkScriptAction extends BaseScriptAction {
       parameterLongName: '--from',
       parameterShortName: '-f',
       argumentName: 'PROJECT2',
-      description: 'Run command in all projects that directly or indirectly depend on the specified project'
+      description: 'Run command in all projects that directly or indirectly depend on the specified project. ' +
```
Comment (sachinjoseph) [inline apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts]: Nit: 1) There are two spaces after the period here, but not in the above change.
2) Can the wording be a bit better? Something like "use . to specify the project in the pwd.".

**Comment by octogonz** [comment]
Finally! 😁

File: apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts (hunk 1 above)
Comment (iclanton) [reply apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts]: 1) Oops. Copy+paste error.
2) Sure. Rephrased a bit.

**Comment by octogonz** [comment]
This was published with Rush 5.20.0

**Pull request merged**